modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/lar/IGDisplayPortletDataHandler.java

```diff
@@ -66,6 +66,7 @@ protected PortletPreferences doDeleteData(
 		portletPreferences.setValue("fileEntryColumns", StringPool.BLANK);
 		portletPreferences.setValue("folderColumns", StringPool.BLANK);
 		portletPreferences.setValue("foldersPerPage", StringPool.BLANK);
+		portletPreferences.setValue("selectedRepositoryId", StringPool.BLANK);
```

Comment:
This has to be sorted alphabetically.

Reply:
I swear that I have a F problem with the r's and the s's.... T_T

Thanks!